script/eval.go

```diff
@@ -60,16 +60,39 @@ func evalSnapshot(r *runner, n node) error {
 func evalUSE(r *runner, n node) error {
```

Comment:
make sure to add unit test for the new usages of SVC_USE

Reply:
in the script package I don't see tests for eval.go, regarding functionality in parser.go and nodes.go I am not sure what else I can add there. For ServiceUse facade method I don't see tests as well.

Reply:
There are tests in parser_test.go. https://github.com/control-center/serviced/blob/develop/script/parser_test.go#L36

It uses a test file to make sure the parsing goes as expected. There are also negative test cases at https://github.com/control-center/serviced/blob/develop/script/parser_test.go#L119

Reply:
added tests